pkg/pdp/aggregator/jobqueue/serializer/serializer.go

```diff
@@ -0,0 +1,29 @@
+package serializer
+
+import (
+	"github.com/ipld/go-ipld-prime/node/bindnode"
+	"github.com/ipld/go-ipld-prime/schema"
+	"github.com/storacha/go-ucanto/core/ipld/codec/cbor"
+)
+
+type Serializer[T any] interface {
```

Comment:
It would be nice if we didn't keep writing IPLD serialize / deserialize mechanism.

we have the Builder interface, the schema parsers, etc. Not your fault, let's just keep it moving

Reply:
Is there a cleaner alternative here? Perhaps this sort of thing should live in go-libstoracha?